concordium-consensus/stack.static.yaml

```diff
@@ -12,4 +12,4 @@ extra-lib-dirs:
 - ./smart-contracts/lib
 
 ghc-options:
-    "$everything": -fPIC
+    "$everything": -fPIC -fsimpl-tick-factor=10000
```

Comment:
Can you please document why this latter option is necessary. And link to the GHC issue https://gitlab.haskell.org/ghc/ghc/-/issues/14637#note_413425 for future maintainers to be able to clean up.

Reply:
done